adm_program/installation/db_scripts/update_3_3.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update>
+    <step id="10">UPDATE %PREFIX%_user_fields SET usf_name_intern = 'STREET' WHERE usf_name_intern LIKE 'ADDRESS'</step>
```

Comment:
thought LIKE should be used with strings and = should be used with int but that was wrong.